py.test fails with existing build dir

My `py.test` was failing with the following error:
```
(nengodev)ctn02:nengo|better-product> py.test                                                                                            ? d31f153795
================================================================ test session starts =================================================================
platform linux2 -- Python 2.7.9 -- py-1.4.26 -- pytest-2.6.4
collected 0 items / 1 errors 

======================================================================= ERRORS =======================================================================
_________________________________________________________________ ERROR collecting . _________________________________________________________________
../../../.pyenv/versions/2.7.9/lib/python2.7/site-packages/py/_path/common.py:334: in visit
    for x in Visitor(fil, rec, ignore, bf, sort).gen(self):
../../../.pyenv/versions/2.7.9/lib/python2.7/site-packages/py/_path/common.py:380: in gen
    for p in self.gen(subdir):
../../../.pyenv/versions/2.7.9/lib/python2.7/site-packages/py/_path/common.py:370: in gen
    if p.check(dir=1) and (rec is None or rec(p))])
../../../.pyenv/versions/2.7.9/lib/python2.7/site-packages/_pytest/main.py:628: in _recurse
    ihook.pytest_collect_directory(path=path, parent=self)
../../../.pyenv/versions/2.7.9/lib/python2.7/site-packages/_pytest/main.py:166: in call_matching_hooks
    plugins = self.config._getmatchingplugins(self.fspath)
../../../.pyenv/versions/2.7.9/lib/python2.7/site-packages/_pytest/config.py:688: in _getmatchingplugins
    plugins += self._conftest.getconftestmodules(fspath)
../../../.pyenv/versions/2.7.9/lib/python2.7/site-packages/_pytest/config.py:521: in getconftestmodules
    mod = self.importconftest(conftestpath)
../../../.pyenv/versions/2.7.9/lib/python2.7/site-packages/_pytest/config.py:545: in importconftest
    raise ConftestImportFailure(conftestpath, sys.exc_info())
E   ConftestImportFailure: (local('/home/jgosmann/Documents/projects/nengo/nengo/tests/conftest.py'), (<class 'py._path.local.ImportMismatchError'>, ImportMismatchError('nengo.tests.conftest', '/home/jgosmann/Documents/projects/nengo/build/lib/nengo/tests/conftest.py', local('/home/jgosmann/Documents/projects/nengo/nengo/tests/conftest.py')), <traceback object at 0x7f5ed0553b48>))
```
After I did `rm -r build` it was working again. I think, the existence of that directory should make no difference (and if it has to, the error message is not helpful at all).

Not sure if it has something to do with me using `pyenv` and `virtualenv`.

--- a/setup.cfg
+++ b/setup.cfg
@@ -13,3 +13,6 @@ ignore =
 
 [upload_sphinx]
 upload-dir = doc/_build/html
+
+[pytest]
+norecursedirs = .* *.egg build dist docs

